submissions/Yasya23/tiny-js-oop/index.js

```diff
@@ -0,0 +1,67 @@
+class Inhabitants {
+  constructor(species, name, gender, saying) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+  }
+
+  showProperties() {
+    return [this.species, this.name, this.gender, this.saying];
+  }
+}
+
+class Human extends Inhabitants {
+  constructor(name, gender, saying) {
+    super("human", name, gender, saying);
+    this.hands = 2;
+    this.legs = 2;
+  }
+
+  showProperties() {
+    return [
+      ...super.showProperties().slice(0, 3),
+      this.hands,
+      this.legs,
+      ...super.showProperties().slice(3, 4),
```

Comment:
I realize why slicing.
Just notice that this is not a really good approach. Why? A: This means that a developer working on `Human` or any other subclass would need to know what is the right order of props in the base class. Likewise, developer working on `Inhabitants` class (e.g. add a property between already existing) should be aware their changes may affect subclasses' behavior. On big projects it is not feasible. Ideally, every class handles its own properties and unaware of other classes internals.

P.S. BTW, class names by convention are singular unless they really contain multiple instances of anything.